analysis/Scripts/NaiveYieldPlot.py

```diff
@@ -148,11 +213,15 @@ def main(argv):
 			
```

Comment:
Think the text in lines 208-212 could be edited (seems to be copy past from plot.py) - and explain what this does a bit better

Reply:
yes agreed.

Will change the text to fit the new code better